__tests__/app.test.js

```diff
@@ -15,102 +15,86 @@ describe('API Routes', () => {
     return client.end();
   });
 
-  const expectedCats = [
-    {
-      id: expect.any(Number),
-      name: 'Felix',
-      type: 'Tuxedo',
-      url: 'cats/felix.png',
-      year: 1892,
-      lives: 3,
-      isSidekick: false
-    },
-    {
-      id: expect.any(Number),
-      name: 'Garfield',
-      type: 'Orange Tabby',
-      url: 'cats/garfield.jpeg',
-      year: 1978,
-      lives: 7,
-      isSidekick: false
-    },
+  const expectedAlbums = [
+
     {
       id: expect.any(Number),
-      name: 'Duchess',
-      type: 'Angora',
-      url: 'cats/duchess.jpeg',
-      year: 1970,
-      lives: 9,
-      isSidekick: false
+      band: 'Quicksand',
+      album: 'Slip',
+      year: 1993,
+      genre: 'Post-Hardcore',
+      img: '',
+      isPlatinum: false
     },
     {
       id: expect.any(Number),
-      name: 'Stimpy',
-      type: 'Manx',
-      url: 'cats/stimpy.jpeg',
-      year: 1990,
-      lives: 1,
-      isSidekick: true
+      band: 'Minor Threat',
+      album: 'Out of Step',
+      year: 1983,
+      genre: 'Punk Hardcore',
+      img: '',
+      isPlatinum: false
     },
     {
       id: expect.any(Number),
-      name: 'Sylvester',
-      type: 'Tuxedo',
-      url: 'cats/sylvester.jpeg',
-      year: 1945,
-      lives: 1,
-      isSidekick: true
+      band: 'X-Ray Spex',
+      album: 'Gremfree Adolescents',
+      year: 1978,
+      genre: 'Punk Rock',
+      img: '',
+      isPlatinum: false
     },
     {
       id: expect.any(Number),
-      name: 'Tigger',
-      type: 'Orange Tabby',
-      url: 'cats/tigger.jpeg',
-      year: 1928,
-      lives: 8,
-      isSidekick: false
+      band: 'Cocks Sparrer',
+      album: 'Shock Troops',
+      year: 1982,
+      genre: 'Punk Rock',
+      img: '',
+      isPlatinum: false
     },
     {
       id: expect.any(Number),
-      name: 'Hello Kitty',
-      type: 'Angora',
-      url: 'cats/hello-kitty.jpeg',
-      year: 1974,
-      lives: 9,
-      isSidekick: false
+      band: 'Hole',
+      album: 'Celebrity Skin',
+      year: 1998,
+      genre: 'Alternative Rock',
+      img: '',
+      isPlatinum: true
     },
     {
       id: expect.any(Number),
-      name: 'Hobbs',
-      type: 'Orange Tabby',
-      url: 'cats/hobbs.jpeg',
-      year: 1985,
-      lives: 6,
-      isSidekick: true
+      band: 'Operation Ivy',
+      album: 'Self-titled',
+      year: 1991,
+      genre: 'Punk Rock',
+      img: '',
+      isPlatinum: false
     }
   ];
+  
 
   // If a GET request is made to /api/cats, does:
   // 1) the server respond with status of 200
   // 2) the body match the expected API data?
-  it('GET /api/cats', async () => {
+  it('GET /api/album', async () => {
```

Comment:
I'm being nitpicky but it's testing /api/albums/ not /album. This is just a label for the test though...so again nitpicky. lol